### ISSUE TITLE
A 32bit processes cannot access modules of a 64bit process

### ISSUE THREAD

**Issue by vorlie** [MManoah/league-practice-tool#6]
As you can see in the screenshot below, i am unable to create 5 stack practice tool lobby due to the application being 32bit meanwhile league client is 64bit. I would ask for a 64bit version if thats possible.

![image](https://github.com/MManoah/league-practice-tool/assets/99135647/d7d6d6c8-48e4-4584-ac60-33bb1fc4b101)

**Comment by pille2002** [comment]
still have the same problem?

**Comment by Fish4Hire** [comment]
Same issue here. 

"As a reminder, support for 32-bit Windows OS will be deprecated on April 4, 2023. After this date a 64-bit version of Windows will be required to play League of Legends and Teamfight Tactics"

**Comment by electricwzrd** [comment]
but is wasn't a problem previously isn't there anyway to accommodate for 32 bit laptops

**Comment by Fish4Hire** [comment]
No. Check your CPU to make sure it's compatible with 64-bit. If it is, reinstall Windows with a 64-bit version. If you're looking or a license, I'd check G2A.
